### ISSUE TITLE
Bug: axe-core/cli validator does not give warning about lack of aria-label or aria-labelledby on role="img" element(ARIA 1.2 )

### ISSUE THREAD

**Issue by spectranaut** [dequelabs/axe-core-npm#316]
**Product:** cli

**Motivation:** 

ARIA 1.2 introduces the following imperative in the [role="img" definition](https://www.w3.org/TR/wai-aria-1.2/#img): `In order for elements with a role of img to be perceivable, authors MUST provide a label using the aria-label or aria-labelledby attribute. `

**Replication of error:** 

If you run the AXE cli (axe-core 4.3.2 in chrome-headless) on an HTML with the following HTML::
```
<div id="img-1" role="img"></div>

<div id="img-2" role="img" title="Image 2 description"></div>

<div id="img-3" role="img">Image 3 description</div>
```
You will see the following errors:
```
  Violation of "role-img-alt" with 2 occurrences!
    Ensures [role='img'] elements have alternate text. Correct invalid elements at:
     - #img-1
     - #img-3
    For details, see: https://dequeuniversity.com/rules/axe/4.3/role-img-alt
```

**Expectation:**

Two errors:
1. You should see an error on all three, because none of the images have an "aria-label" or "aria-labelledby" attribute. 
2. I think the error for this scenario should be under `--tags cat.aria`, but it is not, you see know errors if you run the CLI with `--tags cat.aria`

**Comment by Zidious** [comment]
Thank you for finding this issue! 

I have raised this within axe-core ref: https://github.com/dequelabs/axe-core/issues/3153
